client/components/card/test/index.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+
+/**
+ * Internal dependencies
+ */
+import Card from '../';
+import CompactCard from '../compact';
+
+describe( 'Card', function() {
+	// it should have a class of `card`
+	it( 'should have `card` class', function() {
+		const card = shallow( <Card /> );
+		expect( card.is( '.card' ) ).to.equal( true );
+	} );
+
+	// it should accept a custom class of `test__ace`
+	it( 'should have custom class of `test__ace`', function() {
+		const card = shallow( <Card className="test__ace" /> );
+		expect( card.is( '.card' ) ).to.equal( true );
```

Comment:
@scottsweb just spotted that this should be:

`expect( card.is( '.test__ace' ) ).to.equal( true );`